eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -173,4 +173,29 @@ stages:
                 --toxenv="next-pylint"
                 --disablecov
                 --filter-type="Omit_management"
-            env: ${{ parameters.EnvVars }}
\ No newline at end of file
+            env: ${{ parameters.EnvVars }}
+
+          - task: PythonScript@0
```

Comment:
Is it a goal to run these in every pipeline as this is changing the common test template.

Reply:
No, this should only run for the tests - weekly pipeline: https://github.com/Azure/azure-sdk-for-python/blob/10e361beb0c48563901e93b0eaa99b64128b33d7/eng/pipelines/templates/stages/archetype-sdk-tests.yml#L145

Reply:
Oh great I missed the condition higher up.